src/script/config-diff/README.md

```diff
@@ -18,7 +18,7 @@ python3 config_diff.py <mode> [options]
 
 ### Modes
 
-1. **`diff-branch`**: Compare configuration options between two branches in the same repository.
+1. **`diff-branch`**: Compare configuration options between two branches in the same Ceph repository.
```

Comment:
Fixes to readme/script should be squashed into the previous commit.

Reply:
done!